introduction.md

```diff
@@ -1 +1,10 @@
-I don't really know how to code in Ruby, but I plan on learning soon.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?